src/HealthGPS/runtime_context.h

```diff
@@ -99,7 +103,7 @@ class RuntimeContext {
     std::reference_wrapper<EventAggregator> event_bus_;
     std::reference_wrapper<SimulationDefinition> definition_;
     Population population_;
-    Random generator_;
+    mutable Random generator_;
```

Comment:
Why mutable here?

Reply:
Because that means we can return a non-`const` ref to `generator_` from a `const` method (`random()`). I think it makes sense -- the internal state of your random number generator is not something you actually need to keep `const`. This opens the door for us using `const RuntimeContext &` variables in future.

Reply:
Ah I see. Kinda like saying const method will keep everything constant, besides these few things marked mutable. Thanks,